test/unit/org/apache/cassandra/metrics/TableMetricsTest.java

```diff
@@ -87,31 +95,39 @@ private void executeBatch(boolean isLogged, int distinctPartitions, int statemen
 
         BatchStatement batch = new BatchStatement(batchType);
 
+        for (String table : tables)
+            populateBatch(batch, table, distinctPartitions, statementsPerPartition);
+
+        session.execute(batch);
+    }
+
+    private static void populateBatch(BatchStatement batch, String table, int distinctPartitions, int statementsPerPartition)
+    {
+        PreparedStatement ps = session.prepare(String.format("INSERT INTO %s.%s (id, val1, val2) VALUES (?, ?, ?);", KEYSPACE, table));
+
         for (int i=0; i<distinctPartitions; i++)
         {
             for (int j=0; j<statementsPerPartition; j++)
             {
                 batch.add(ps.bind(i, j + "a", "b"));
             }
         }
-
-        session.execute(batch);
     }
 
     @Test
     public void testRegularStatementsExecuted()
     {
         ColumnFamilyStore cfs = recreateTable();
         assertEquals(0, cfs.metric.coordinatorWriteLatency.getCount());
-        assertTrue(cfs.metric.coordinatorWriteLatency.getMeanRate() == 0);
+        assertEquals(0.0, cfs.metric.coordinatorWriteLatency.getMeanRate(), 0.0);
```

Comment:
the reason for this change is `assertTrue` has really bad error messages; with `assertEquals` we know what the mean rate will be